sources/commands/Project.ts

```diff
@@ -0,0 +1,46 @@
+import {Command, UsageError} from 'clipanion';
+import semverValid           from 'semver/functions/valid';
+import semverValidRange      from 'semver/ranges/valid';
+
+import {BaseCommand}         from './Base';
+
+// modified from ./Enable.ts
+// https://github.com/nodejs/corepack/issues/505
+export class ProjectInstallCommand extends BaseCommand {
+  static paths = [
+    [`project`, `install`],
+  ];
+
+  static usage = Command.Usage({
+    description: `Add the Corepack shims to the install directories, and run the install command of the specified package manager`,
+    details: `
+      When run, this command will check whether the shims for the specified package managers can be found with the correct values inside the install directory. If not, or if they don't exist, they will be created.
+
+      Then, it will run the install command of the specified package manager. If no package manager is specified, it will default to NPM.
```

Comment:
```suggestion
      Then, it will run the install command of the specified package manager. If no package manager is specified, it will default to npm.
```